lib/directory.coffee

```diff
@@ -30,8 +32,6 @@ class Directory
     @status = null
     @entries = {}
 
-    @submodule = repoForPath(@path)?.isSubmodule(@path)
```

Comment:
Are we eliminating the `submodule` distinction with the new repository API or has it just moved elsewhere?